### DISCUSSION TITLE
Xenforo Plugin

### DISCUSSION THREAD

**Discussion by BenjaminM007** [btcpayserver/btcpayserver#6126] [Plugins & integrations]
We have noticed that when a customer pays with BTCPay and they over pay the invoice the payment is not added to their account. However, we have notieced two things.

If we manually mark the payment as settled in BTCPay the webhook instantly fails with a status of a red X and no other callback will work on any future payments unless we removed the webhook and add it again.

In addition to this. Even when the payment is manually changed to settled the BTCPay plugin in Xenforo cannot find the transaction.

     _`Error: Could not validate purchasable: Unable to find transaction '25067'`_ 

Below is the full error log in Xenforo minus some edited keys

`array(3) {
  ["signature"] => string(71) "sha256=9c8b026d6XXXXXXXXXXXX0f3983205cf714438f5ad6508bac993e44580f"
  ["type"] => string(14) "InvoiceSettled"
  ["body"] => string(472) "{
  "deliveryId": "XGCABCXXXXXXXXXbWA2xZc",
  "webhookId": "BMu7LCXXXXXXXXXXX6rVCz",
  "originalDeliveryId": "4t2L2VXXXXXXspHw9w",
  "isRedelivery": true,
  "type": "InvoiceSettled",
  "timestamp": 1722213086,
  "manuallyMarked": true,
  "overPaid": false,
  "storeId": "CH3P2hYfLL3XXXXXXXXXXXXXSvsjyzGpSPJTYa",
  "invoiceId": "GQHsXXXXXXXXXKEHDgD",
  "metadata": {
    "request_key": "KYJjkR24XXXXXXXXXXXOxH46c7oe",
    "purchase_request_id": 18056
  }
}"
}`

We are using Xenforo 2.2.16 and the latest BTCPay  © BTCPay Server v1.13.5+30d91d0bd  BTCPay Provider 1.0.0 ( Xenforo Plugin )

Thank you

**Comment by pavlenex** [comment]
I've opened this into issue on XenForo repo, so the author can take a look once he gets a chance. https://github.com/btcpayserver/xenforo/issues/7

**Reply by BenjaminM007** [reply]
Ok Thank you, I appreciate it.
